pages/blog/changes-coming-docs.md

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
@derberg I seem to have lost one of my images ( the mind map), so I will be making it again and attaching tmw 😬

Reply:
@derberg pingo pingo! 😂✨✨

Added imgs and updated post to mention TSC.

Reply:
you need to check out the preview of the article, some images are broken
![Screenshot 2021-12-06 at 10 17 56](https://user-images.githubusercontent.com/6995927/144820166-c3bab3c5-ee18-402a-8c68-8b42a4581c72.png)

Also please have a look at other articles and their metadata and use it properly here:
- what is `Subtitle` metadata?
- `date` is pretty much used differently, in the UI it looks like you wrote article in 1970 😆 
- please add `featured: true` so it will show at the beginning of the list for some time

Reply:
Hey! This is not helpful. I told you that I needed help understanding the metadata, and that I had done the best I could from looking at other articles.

Can you post a helpful comment that explains these points instead of assuming I get them? :)

Reply:
Ooh! I see now, this is making more sense now. Thank you because I really needed that extra clarification because I felt very lost. OK, I will fix it in my next commit!!! ✨✨✨✨